src/SFA.DAS.ApprenticeFeedback.Web/Pages/ExitSurvey/CheckYourAnswers.cshtml

```diff
@@ -112,20 +115,34 @@
                             </a>
                         </dd>
                     </div>
+
+                    <div class="govuk-summary-list__row">
+                        <dt class="govuk-summary-list__key govuk-!-width-one-third">
+                            What did you do after leaving your apprenticeship?
+                        </dt>
+                        <dd class="govuk-summary-list__value">
+                            @Model.ExitSurveyContext.Attributes.FirstOrDefault(a => a.Category == SFA.DAS.ApprenticeFeedback.Domain.Models.ExitSurvey.ExitSurveyAttributeCategory.PostApprenticeshipStatus)?.Name
+                        </dd>
+                        <dd class="govuk-summary-list__actions">
+                            <a class="govuk-link--no-visited-state" href="/exit/question4">
+                                Change<span class="govuk-visually-hidden"> question 4</span>
```

Comment:
These change links are 'question 1', 'question 2' 'primary reason', 'question 3' and 'question 4' - run this by content

Reply:
I discussed the content change with the BA, and they have confirmed that it is acceptable